src/commands/dev/dev.js

```diff
@@ -11,6 +11,8 @@ const stripAnsiCc = require('strip-ansi-control-characters')
 const waitPort = require('wait-port')
 
 const { startFunctionsServer } = require('../../lib/functions/server')
+const { ensureAppForSite, startOneGraphCLISession } = require('../../lib/oneGraph/client')
```

Comment:
NIT: We have the convention in the codebase to use kebab-case namings for folders over camelCase so please change for consistency.

```suggestion
const { ensureAppForSite, startOneGraphCLISession } = require('../../lib/one-graph/client')
```

Reply:
Done!